device/mellanox/x86_64-mlnx_msn4410-r0/ACS-MSN4410/buffers.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers.json.j2
```

Comment:
it should be the same as sn4700. the buffers are different for spc and spc3

Reply:
[ACS-MSN4700/buffers.json.j2](https://github.com/Azure/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/buffers.json.j2)

It is the same, please check the link bellow